src/bionics.cpp

```diff
@@ -1734,6 +1734,15 @@ void Character::process_bionic( int b )
     }
 }
 
+void Character::roll_critical_bionics_failure( body_part bp )
+{
+    const hp_part limb = bp_to_hp( bp );
+
+    if( one_in( hp_cur[limb] ) / 4 ) {
```

Comment:
If true/false over 4?

Reply:
Wow yikes, thanks for catching that! I have placed that 4 inside the one_in()

Reply:
I'm surprised none of our CI did